Formula/percona-server.rb

```diff
@@ -8,8 +8,15 @@ class PerconaServer < Formula
   revision 2
 
   livecheck do
-    url "https://www.percona.com/downloads/Percona-Server-LATEST/"
-    regex(/value=.*?Percona-Server[._-]v?(\d+(?:\.\d+)+-\d+)["' >]/i)
+    url "https://docs.percona.com/percona-server/latest/index.html"
```

Comment:
```suggestion
    url "https://docs.percona.com/percona-server/latest/"
```

I realized right after you committed the suggested changes that we can omit `index.html` from the end (though the trailing forward slash is still required here). It's functionally the same either way but I try to leave off `index.html` when it's not required, just for the sake of simplicity. If it's not too much trouble, we may want to update this and `percona-xtrabackup` one last time before merging.